odmlui/treemodel/GenericIter.py

```diff
@@ -41,7 +41,7 @@ def get_next(self):
         returns a new Iter object for the next element in this multivalue list
         or None
         """
-        obj = next(self._obj)
+        obj = self._obj.__next__()
```

Comment:
The line being deleted here (red one) was introduced by me, in the starting commits of this PR. :sweat_smile: 
Initially, it was `self._obj.next()` which works fine in Python 2, but has issues in Python 3. 
Python 3 requires the `next(self._obj)` method. So, the `next()` method worked in Py 3, but not in Py 2. Hence, in this commit, I changed it to the most specific function name. 
In Python 2, to make `next()` work, it seems that we need to implement a `next()` method in the object, and not `__next__()`. So, to just have the standard `next()` method and be Py2/3 compatible, we need to alias `need` to `__need__` method in the `nodes.py` in odML core library. So, instead of making a change there, I made the change here. 
If you feel that the `next()` method is more suitable, do let me know. I will make the change in the `python-odml` repo, and use `next()` method here.